src/common/components/ui/formik/RemovableField.css

```
@@ -0,0 +1,9 @@
+.remove-button--tooltip {
```

I know this is not yet used widely, but we should try to follow BEM and ideally prefix all our rules with `ai`
```suggestion
.ai-remove-button__tooltip {
```